portfolio/src/main/java/com/google/sps/servlets/LoadImagesServlet.java

```diff
@@ -45,8 +45,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         List < Entity > resultsList = results.asList(FetchOptions.Builder.withDefaults());
         List < Image > images = new ArrayList < > ();
 
-        for (int i = 0; i < resultsList.size(); i++) {
-            Entity entity = resultsList.get(i);
+        for (int result = 0; result < resultsList.size(); result++) {
```

Comment:
It's still index not the result. It's nice to name variables like that when you do for(result : resultsList) {} Make it back to i

Reply:
done!